ISISComputingGroup/EPICS-ioc
# Ticket 3230: Added support for different CCD100 addresses

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/3230

Requires:
* Support module: https://github.com/ISISComputingGroup/EPICS-CCD100/pull/4
* Emulator changes: https://github.com/ISISComputingGroup/EPICS-DeviceEmulator/pull/64
* System tests: https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/98

### To test

* Run the associated system tests
* Confirm that you can set the address macro of the CCD100 IOC and it sends the appropriate strings to the device
* Confirm the address defaults to a

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
